pkg/plugins/git/generic/main.go

```diff
@@ -126,15 +152,31 @@ func Checkout(branch, remoteBranch, workingDir string) error {
 		})
 
 		if err != nil {
+			logrus.Debugln(err)
 			return err
 		}
-	}
 
-	logrus.Infof("")
+		err = w.Checkout(&git.CheckoutOptions{
```

Comment:
```suggestion
		_ = w.Checkout(&git.CheckoutOptions{
```

(or add a `if err!:nil` block after).
Reported by the linter:

```
pkg/plugins/git/generic/main.go:159:3: ineffectual assignment to `err` (ineffassign)
                err = w.Checkout(&git.CheckoutOptions{
                ```